gloo/docs/user_guides/advanced_routing_action.md

```diff
@@ -116,7 +116,84 @@ glooctl add route \
   --rest-parameters ':path=/petstore/findPetById/{id}'
 ```
 
-<!-- TODO: add doc on Subset -->
+#### Subset
+
+[Subset]({{% ref "/v1/github.com/solo-io/gloo/projects/gloo/api/v1/subset.proto.sk#subset" %}}) currently lets you
+provide a Kubernetes selector to allow request forwarding to a subset of Kubernetes Pods within the upstream associated
```

Comment:
technically this is not specific to kubernetes;
i.e. i plan to have this working also for consul